plugins/workspace-backpack/src/backpack.js

```diff
@@ -497,6 +497,26 @@ export class Backpack extends Blockly.DragTarget {
     return JSON.stringify(json);
   }
 
+  /**
+   * Converts serialized XML to its equivalent serialized JSON string
+   * @param {!string} blockXml The XML serialized block.
+   * @returns {string} The JSON object as a string.
+   * @private
+   */
+  blockXmlToJsonString(blockXml) {
+    if (!blockXml.startsWith('<block')) {
+      throw new Error('Unrecognized XML format');
+    }
+
+    const workspace = new Blockly.Workspace();
```

Comment:
Can you call `.dispose` on this at the end of the method to make sure we're not leaking any state? E.g. workspaces get registered so they can be retrieved with `Blockly.getWorkspaceById`, and disposing removes that registration.